==============================================================================
Productor                                                                     
==============================================================================
Productor.Testcase                                                            
==============================================================================
[ERROR] EXCEPTION RAISED DURING PABOT EXECUTION
[ERROR] PLEASE CONSIDER REPORTING THIS ISSUE TO https://github.com/mkorpela/pabot/issues
Pabot: 2.18.0
Python: 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)]
Robot Framework: 6.0.2
Total testing: 0.0 seconds
Elapsed time:  0.0 seconds
Error: Command 'pabot -d .\output\Testcase  -i TC02 .' returned non-zero exit status 1.
Productor.Testcase.TC02                                                       
==============================================================================
Category                                                              | PASS |
------------------------------------------------------------------------------
Productor.Testcase.TC02                                               | PASS |
1 test, 1 passed, 0 failed
==============================================================================
Productor.Testcase                                                    | PASS |
1 test, 1 passed, 0 failed
==============================================================================
Productor                                                             | PASS |
1 test, 1 passed, 0 failed
==============================================================================
Output:  C:\Productor\output\Testcase\pabot_results\0\output.xml
